src/index.js

```diff
@@ -57,9 +62,39 @@ const parseText = (text = '', fn = () => {}) => {
     let [substring, isClosing, tag] = match;
     let index = match['index'];
     tag = (tag || '').trim();
-    isClosing = (isClosing || '').trim();
     substring = (substring || '').trim();
-    if (!isClosing) {
+    isClosing = (isClosing || '').trim();
+    let selfClosing = voidTags.includes(tag) || /<([^\/>]+)\/>/.test(substring);
+
+    if (selfClosing) {
+      let attr = substring.replace(/[<>]/gi, '').replace(tag, '');
+      let props = extractPropsFromString(attr);
+      let JsxEl = fn(tag, { ...props });
+
+      JsxEl = cloneElement(
+        JsxEl,
+        ''
+      );
+
+      nodes = [...nodes];
```

Comment:
What's the purpose of this? Isn't nodes already an array? For React children I think `React.Children.toArray(children)` should be used.

Reply:
Good question. I will answer once I investigate the difference. I based it on existing pieces of code.